x-pack/plugins/observability_solution/infra/public/components/logging/log_minimap/time_ruler.tsx

```diff
@@ -60,7 +60,7 @@ export const TimeRuler: React.FC<TimeRulerProps> = ({ end, height, start, tickCo
 TimeRuler.displayName = 'TimeRuler';
 
 const TimeRulerTickLabel = styled.text`
-  font-size: 9px;
+  font-size: ${(props) => useEuiFontSize('xxxs').fontSize};
```

Comment:
The same case here and other places with this usage.

Reply:
thanks